src/components/dialogs/input-hooks.js

```diff
@@ -680,9 +681,18 @@ export const useExpandableValues = ({
     fieldProps,
     validateItem,
 }) => {
+    const useCostumStyle = makeStyles((theme) => ({
+        errorItem: {
+            color: theme.palette.error.main,
+            textDecorationLine: 'overline',
+            margin: theme.spacing(1),
+        },
+    }));
+    const costumClasses = useCostumStyle();
     const classes = useStyles();
     const [values, setValues] = useState([]);
     const [errors, setErrors] = useState();
+    const [errorItem, setErrorItem] = useState(false);
```

Comment:
something like : [isValuesListEmply, setValuesListEmpty] ?
This error is specific to the empty list problem

Reply:
okey ill use [emptyListError, setEmptyListError]